### DISCUSSION TITLE
Bind color for ui.icon

### DISCUSSION THREAD

**Discussion by oxbown** [zauberzeug/nicegui#2684] [Ideas / Feature Requests]
The idea is to bind the color prop of an icon to a variable. 

```py
ui.icon('circle', color='grey').classes('text-2xl').bind_color_from(input1, 'value', backward=lambda n: "green" if n is True else "red"')
```

**Comment by falkoschindler** [comment]
Thanks for the suggestion, @oxbown!
But I'll need to close it as a duplicate, since this feature is already being discussed over here: https://github.com/zauberzeug/nicegui/discussions/1389.
